packages/react-button/src/components/MenuButton/useMenuButtonStyles.ts

```diff
@@ -1,52 +1,31 @@
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
-import { Theme } from '@fluentui/react-theme';
 import { useButtonStyles } from '../Button/useButtonStyles';
-import { MenuButtonState, MenuButtonStyleSelectors, MenuButtonVariantTokens } from './MenuButton.types';
+import { MenuButtonState } from './MenuButton.types';
 
-export const makeMenuButtonTokens = (theme: Theme): MenuButtonVariantTokens => ({
-  base: {
-    menuIconFontSize: '20px',
-    menuIconHeight: '20px',
-    menuIconWidth: '20px',
+const useMenuIconStyles = makeStyles({
+  small: {
+    fontSize: '20px',
+    height: '20px',
+    width: '20px',
   },
-  large: {
-    menuIconFontSize: '24px',
-    menuIconHeight: '24px',
-    menuIconWidth: '24px',
-  },
-});
-
-const useStyles = makeStyles({
-  menuIcon: theme => {
-    const menuButtonTokens = makeMenuButtonTokens(theme);
-
-    return {
-      fontSize: menuButtonTokens.base?.menuIconFontSize,
-      height: menuButtonTokens.base?.menuIconHeight,
-      width: menuButtonTokens.base?.menuIconWidth,
-    };
+  medium: {
+    fontSize: '20px',
+    height: '20px',
+    width: '20px',
   },
-  menuIconLarge: theme => {
-    const menuButtonTokens = makeMenuButtonTokens(theme);
-
-    return {
-      fontSize: menuButtonTokens.large?.menuIconFontSize,
-      height: menuButtonTokens.large?.menuIconHeight,
-      width: menuButtonTokens.large?.menuIconWidth,
-    };
+  large: {
+    fontSize: '24px',
+    height: '24px',
+    width: '24px',
   },
 });
 
-export const useMenuButtonStyles = (state: MenuButtonState, selectors: MenuButtonStyleSelectors) => {
+export const useMenuButtonStyles = (state: MenuButtonState): MenuButtonState => {
   useButtonStyles(state);
```

Comment:
Similar to CompoundButton, should `useButtonStyles` be called at the end of this function so that the styles defined here will override the button styles? (Probably not an issue now, but in the future someone might try to add more styles and get confused why it's not working as expected).

Reply:
Yeah, will make the changes.